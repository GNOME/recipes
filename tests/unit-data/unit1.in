gram
g
kilogram
kg
lb
pound
ounce
oz
liter
l
deciliter
dl
milliliter
ml
fluid ounce
fl oz
pint
pt
quart
qt
gallon
gal
cup
tablespoon
tbsp
teaspoon
tsp
box
package
pkg
