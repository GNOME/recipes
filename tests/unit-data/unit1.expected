INPUT 'gram'
REST ''
UNIT g
NAME gram
ABBREVIATION g

INPUT 'g'
REST ''
UNIT g
NAME gram
ABBREVIATION g

INPUT 'kilogram'
REST ''
UNIT kg
NAME kilogram
ABBREVIATION kg

INPUT 'kg'
REST ''
UNIT kg
NAME kilogram
ABBREVIATION kg

INPUT 'lb'
REST ''
UNIT lb
NAME pound
ABBREVIATION lb

INPUT 'pound'
REST ''
UNIT lb
NAME pound
ABBREVIATION lb

INPUT 'ounce'
REST ''
UNIT oz
NAME ounce
ABBREVIATION oz

INPUT 'oz'
REST ''
UNIT oz
NAME ounce
ABBREVIATION oz

INPUT 'liter'
REST ''
UNIT l
NAME liter
ABBREVIATION l

INPUT 'l'
REST ''
UNIT l
NAME liter
ABBREVIATION l

INPUT 'deciliter'
REST ''
UNIT dl
NAME deciliter
ABBREVIATION dl

INPUT 'dl'
REST ''
UNIT dl
NAME deciliter
ABBREVIATION dl

INPUT 'milliliter'
REST ''
UNIT ml
NAME milliliter
ABBREVIATION ml

INPUT 'ml'
REST ''
UNIT ml
NAME milliliter
ABBREVIATION ml

INPUT 'fluid ounce'
REST ''
UNIT fl oz
NAME fluid ounce
ABBREVIATION fl oz

INPUT 'fl oz'
REST ''
UNIT fl oz
NAME fluid ounce
ABBREVIATION fl oz

INPUT 'pint'
REST ''
UNIT pt
NAME pint
ABBREVIATION pt

INPUT 'pt'
REST ''
UNIT pt
NAME pint
ABBREVIATION pt

INPUT 'quart'
REST ''
UNIT qt
NAME quart
ABBREVIATION qt

INPUT 'qt'
REST ''
UNIT qt
NAME quart
ABBREVIATION qt

INPUT 'gallon'
REST ''
UNIT gal
NAME gallon
ABBREVIATION gal

INPUT 'gal'
REST ''
UNIT gal
NAME gallon
ABBREVIATION gal

INPUT 'cup'
REST ''
UNIT cup
NAME cup
ABBREVIATION cup

INPUT 'tablespoon'
REST ''
UNIT tbsp
NAME tablespoon
ABBREVIATION tbsp

INPUT 'tbsp'
REST ''
UNIT tbsp
NAME tablespoon
ABBREVIATION tbsp

INPUT 'teaspoon'
REST ''
UNIT tsp
NAME teaspoon
ABBREVIATION tsp

INPUT 'tsp'
REST ''
UNIT tsp
NAME teaspoon
ABBREVIATION tsp

INPUT 'box'
REST ''
UNIT box
NAME box
ABBREVIATION box

INPUT 'package'
REST ''
UNIT pkg
NAME package
ABBREVIATION pkg

INPUT 'pkg'
REST ''
UNIT pkg
NAME package
ABBREVIATION pkg

